config/crd/kustomization.yaml

```diff
@@ -2,9 +2,10 @@
 # since it depends on service name and namespace that are out of this kustomize package.
 # It should be run by config/default
 resources:
-  - bases/workload.nephio.org_amfdeployments.yaml
-  - bases/workload.nephio.org_smfdeployments.yaml
-  - bases/workload.nephio.org_upfdeployments.yaml
+        #  - bases/workload.nephio.org_amfdeployments.yaml
```

Comment:
Do we need to keep the commented out code?

Reply:
NO.. we will remove the comments.